src/Aquifer.Jobs/appsettings.example.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
If this PoC works (I'm assuming it does) then we should move to only using the `appsettings.json` paradigm.  Thus, everything in `local.settings.example.json` should move into this new `appsettings.example.json`, instead, as part of this PR.